### DISCUSSION TITLE
GitHub.dev (Web Editor) does not show build errors

### DISCUSSION THREAD

**Discussion by IonicPixels** [community/community#12818] [Other Feature Feedback, Questions, & Ideas]
I've been using the web editor for a while while I am away from my computer. I've noticed that the web editor does not show build errors like Visual Studio does, where every issue with the code will have a red underline. It would be nice to have a feature to view errors so its easier to debug, especially when writing over 300 lines of code. I've dont this, and ended up breaking the code with 20+ errors.

**Comment by Emasoft** [comment]
We need a bunch of Linters for the web editor, since the desktop extensions do not work.
